src/main/java/mate/academy/service/BookService.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.service;
+
+import java.util.List;
+import java.util.Optional;
+import mate.academy.entity.Book;
+
+public interface BookService {
+    Book createBook(Book book);
+
+    Optional<Book> getBookById(int id);
+
+    List<Book> findAllBooks();
+
+    Book updateBook(int id,Book book);
+
+    boolean deleteBookById(int id);
```

Comment:
The `deleteBookById` method should use `Long` for the `id` parameter instead of `int`, as specified in the task requirements.